cache-refresh.R

```diff
@@ -50,8 +50,9 @@ get_data_for_season <- function(season) {
     rename(site_id = id)
   
   cultivars_table <- tbl(bety_src, 'cultivars') %>%
-    rename(cultivar_id = id, cultivar_name = name) %>%
-    select(cultivar_id, cultivar_name)
+    rename(cultivar_id = id) %>% 
```

Comment:
it is not clear why this would fix a bug; one thing I can imagine is that you have loaded another package with a conflicting namespace (which is possible, since there are many versions of a function called 'rename'. 

What was the original error? Does using `dplyr::rename(cultivar_id = id, cultivar_name = name)` resolve the issue? If so, that would be the preferred resolution.